src/Compilers/Core/Portable/CommandLine/SarifV2ErrorLogger.cs

```diff
@@ -280,6 +296,15 @@ private void WriteRules()
                             _writer.WriteArrayEnd(); // suppressionKinds
                         }
 
+                        if (reportAnalyzerExecutionTime)
+                        {
+                            var executionTime = ReportAnalyzerUtil.GetFormattedAnalyzerExecutionTime(descriptorInfo.ExecutionTime, _culture).Trim();
+                            _writer.Write("executionTimeInSeconds", executionTime);
+
+                            var executionPercentage = ReportAnalyzerUtil.GetFormattedAnalyzerExecutionPercentage(descriptorInfo.ExecutionPercentage, _culture).Trim();
```

Comment:
Why `.Trim()` these two values? Are we using `.Trim()` elsewhere?

Reply:
These values include spaces for alignment in the `/reportanalyzer` output: https://github.com/dotnet/roslyn/blob/main/docs/analyzers/Report%20Analyzer%20Format.md#example. We want to avoid emitting these spaces in the SARIF log